spec/ruby/core/string/uminus_spec.rb

```diff
@@ -3,4 +3,8 @@
 
 describe 'String#-@' do
   it_behaves_like :string_dedup, :-@
+
+  ruby_version_is "3.2" do
+    it_behaves_like :string_dedup, :dedup
+  end
```

Comment:
**minor:** It's already tested in a dedicated spec file `spec/ruby/core/string/dedup_spec.rb`

Reply:
Ups, missed that, thanks.